auth-middleware-test-svc/flaskapp.py

```diff
@@ -12,8 +12,16 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+# TODO manage the import when commit and push
+# use this for installed auth library
+# import flask
+# import auth_middleware
+
+# use this for development without installing auth library
+# changed the name lib/auth-middleware to lib/auth_middleware.
+# Change it back when push
 import flask
-import auth_middleware
+import lib.auth_middleware.auth_middleware as auth_middleware
```

Comment:
This needs to be reverted, right? The folder name and import statement?

Reply:
sorry, reverted

Reply:
No issues. Thanks.